DispatcherService/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:8
+
+RUN wget https://yarnpkg.com/latest.tar.gz
```

Comment:
it comes with yarn pre-installed . check https://github.com/nodejs/docker-node#image-variants